DirectEmployers/MyJobs
# Added final requested column. [PD-3077]

## What's this PR do?
Adds the last required field for the site analytics and job analytics report. 

## Screenshots (if appropriate)
![screen shot 2017-04-21 at 9 16 17 am](https://cloud.githubusercontent.com/assets/2728248/25284355/be48fd8a-2684-11e7-90b9-b7a532df431d.png)

## Where should the reviewer start?
Logic was all already there, this just enables the column and fixes the tests that look for it. 

## How should this be manually tested?
Navigate to the site or job analytics report. The Source/ATS Apply Rate column should be present and populated.

## Any background context you want to provide?
The % in that column makes no sense because our make_data isn't random and nonsensical. Percentages are much, much lower using production data.
